### PR TITLE
Version label was almost not visible

### PR DESCRIPTION
**Pull request by tymmesyde**

### CODE REVIEW

File: src/routes/Settings/styles.less [hunk 1]
```diff
@@ -69,7 +69,8 @@
             .version-info-label {
                 flex: 0 1 auto;
                 margin: 0.5rem 0;
-                color: var(--overlay-color);
+                color: var(--primary-foreground-color);
+                opacity: 0.5;
```
Comment (gaurovgiri) [inline src/routes/Settings/styles.less]:
```suggestion
                opacity: 0.3;
```

**Review by gaurovgiri — CHANGES_REQUESTED** [review]
Setting opacity to 0.3 will help distinguish the version info from the clickable buttons in settings which are of opacity 0.4.

**Review by kKaskak — CHANGES_REQUESTED** [review]
It looks good with opacity 0.3, but it might require display changes on mobile. As it's brighter than before and fixed with the Navbar. What if we just put it in the same place as before but not fixed, visible only when you scroll down the settings and add some margin after it, so there is enough spacing between the label and the Navbar.

**Comment by gaurovgiri** [comment]
@kKaskak Yeah, I think doing so might be better.

File: src/routes/Settings/styles.less [hunk 2]
```diff
@@ -465,6 +465,10 @@
                         }
                     }
                 }
+
```
Comment (kKaskak) [inline src/routes/Settings/styles.less:468]: @tymmesyde You can remove this as it works fine without specifying flex in the minimum size too 👍

File: src/routes/Settings/styles.less [hunk 3]
```diff
@@ -438,18 +449,7 @@
             flex-direction: column-reverse;
 
             .side-menu-container {
```
Comment (kKaskak) [inline src/routes/Settings/styles.less:451]: remove this too as its already specified with display: none on a higher resolution size

**Review by kKaskak — CHANGES_REQUESTED** [review]

File: src/routes/Settings/styles.less (hunk 2 above)
Comment (kKaskak) [reply src/routes/Settings/styles.less:468]: 468-471

File: src/routes/Settings/styles.less (hunk 3 above)
Comment (kKaskak) [reply src/routes/Settings/styles.less:451]: 451-453

Comment (tymmesyde) [reply src/routes/Settings/styles.less:451]: It's better to make it clear what do we want to change for each screen sizes

**Review by kKaskak — APPROVED** [review]

**Pull request merged**